packages/@okta/vuepress-site/use_cases/inline_hooks/index.md

```diff
@@ -44,11 +46,11 @@ The graphic below illustrates the sequence of steps:
 
 1. During the execution of an Okta process flow, at the extension point between points A and B, Okta sends a request to your external service.
 
-1. Your external service performs some processing.
+2. Your external service performs some processing.
 
-1. Your external service sends a response back to Okta.
+3. Your external service sends a response back to Okta.
 
-1. Okta receives the response, acts on any commands it includes, and resumes the process flow that originally triggered the inline hook.
+4. Okta receives the response, acts on any commands it includes, and resumes the process flow that originally triggered the inline hook.
```

Comment:
Keeping these all as `1` is actually more robust, because VuePress does the numbering automatically anyway, and this way you can add a step without having to renumber everything.

Reply:
Ack, my vscode extension got too greedy. Wasn't supposed to modify this.